dis_snek/client/utils/formatting.py

```diff
@@ -0,0 +1,113 @@
+import enum
+from typing import Optional
+
+
+def bold(text: str) -> str:
+    """Formats text for discord message as bold"""
+    return f"**{text}**"
+
+
+def italic(text: str) -> str:
+    """Formats text for discord message as italic"""
+    return f"*{text}*"
+
+
+def underline(text: str) -> str:
+    """Formats text for discord message as underlined"""
+    return f"__{text}__"
+
+
+def strikethrough(text: str) -> str:
+    """Formats text for discord message as strikethrough"""
+    return f"~~{text}~~"
+
+
+def spoiler(text: str) -> str:
+    """Formats text for discord message as spoiler"""
+    return f"||{text}||"
+
+
+def no_embed_link(url: str) -> str:
+    """Makes link in discord message display without embedded website preview"""
+    return f"<{url}>"
+
+
+def link_in_embed(text: str, url: str) -> str:
+    """Makes a clickable link inside embed"""
+    return f"[{text}]({url})"
+
+
+def quote_line(line: str) -> str:
+    """Formats a text line for discord message as quote"""
+    return f">{line}"
```

Comment:
As far I know Discord wants a SPACE between `>` and `QUOTED TEXT`

Reply:
Good catch